driftdetect/driftdetectortype.py

```diff
@@ -0,0 +1,10 @@
+#
+# Assumption drift detection based on baseline and intel service graph data
+#
+# Author: Sacha Faust (sfaust@lyft.com)
+#
+from enum import IntEnum
+
+
+class DriftDetectorType(IntEnum):
```

Comment:
Move this class into the `driftdetector` module and delete this module entirely.